### PR TITLE
Build /security/fips page

### LINKED ISSUE #9749: [security certifications] Build /security/fips page
- [x] FIPS page (/security/fips) [copy doc](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)

Please use feature branch security-certifications - (Make PR against this instead of master)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build new page at `/security/fips`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/fips
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc](https://docs.google.com/document/d/1moTJCAtFXKD7ZXrxB-EB7GPVyZRTSIdkP-0BhPZl69M/edit#)

## Issue / Card

Fixes #9749

### CODE REVIEW

File: templates/security/fips.html [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+      </tbody>
+    </table>
+    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140-2 validated modules</a>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h3>Canonical patches vulnerabilities</h3>
+    <p>Each FIPS 140-2 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/fips">our security pages on how to enable these options</a>.</p>
```
Comment (petesfrench) [inline templates/security/fips.html:172]:
```suggestion
    <p>Each FIPS 140-2 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/fips">our security pages on how to enable these options</a>.</p>
```
Change diagonal apostrophes to straight ones

File: templates/security/fips.html [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    <p>We recommend enabling the 'fips-updates' option that includes the security fixes. Our cloud images starting with 20.04 are pre-configured to enable security updates. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance phase</a>.</p>
```
Comment (petesfrench) [inline templates/security/fips.html:173]:
```suggestion
    <p>We recommend enabling the 'fips-updates' option that includes the security fixes. Our cloud images starting with 20.04 are pre-configured to enable security updates. The packages from 'fips-updates' option are updated to include high and critical security fixes during the whole product lifecycle including the <a href="/security/esm">Extended Security Maintenance phase</a>.</p>
```
Change diagonal apostrophes to straight ones

**Review by petesfrench — COMMENTED** [review]
All good :+1: Just some minor changes in one section.

**Comment by anasereijo** [comment]
Some suggestions/questions:

- [ ] as the URL is `/security/fips` should this page have a sec. nav?

- [ ] on https://ubuntu.com/security/certifications#fips there's a section about Fips. I think this should be updated with a link to this new page. @SirSamTumless I left a comment in the [copydoc](https://docs.google.com/document/d/1CiF-dPqG1Y--h4FWfgKB1ft4G3nC2Boa8NT_JG7D6dQ/edit#heading=h.9xddh331a12x) and I tagged you as I don't know who's the best person to ask this.

I also asked @petesfrench the above questions in regards to [/security/cis](https://github.com/canonical-web-and-design/ubuntu.com/issues/9850#issuecomment-852942854). These pages should be synced regarding those suggestions.

More suggestions:

- [ ] I updated some headings in the doc. please update it accordingly 
- [ ] on 'Access FIPS images on the cloud' you currently have a table but I think it will work nicer if we do something as we have for the [raspberry-pi boards](https://ubuntu.com/download/raspberry-pi)
- [ ] can we have hr separating the followin sections?
Screenshot 2021-06-02 at 15 36 51

Screenshot 2021-06-02 at 15 37 36

**Comment by bethcollins92** [comment]
Thanks @anasereijo, I've made those changes (going to double check about the secondary nav)

**Comment by anasereijo** [comment]
looking good, thanks! but you forgot to add the hr here:

Screenshot 2021-06-02 at 15 37 36

**Comment by bethcollins92** [comment]
@anasereijo wups, added!

**Comment by anasereijo** [comment]
thanks! +1

**Pull request merged**